src/index.js

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env node
+
+// Register means every single file will get processed by babel that we import!
+import 'babel/register'
+
+import yaml from 'js-yaml';
+import fsPath from 'path';
+import fs from 'mz/fs';
+import createCommands from './commands';
+
+const CONFIG = 'exhibition.yml';
```

Comment:
Should this not be relative to the `$ROOT` from the `exhibition` or do you want to traverse up the folder hierarchy.

Note, the normal way of doing this with node is to require people to do `npm install -g exhibition-cli` and then all that does is walk up the folder hierarchy from cwd and find the config file and do things... Kinda of how grunt works, except it doesn't walk up the folder hierarchy. But it's global install is just a very minimal script that ends up in PATH and works it's magic from there.

I know we like things to be self contained, but node/npm global install actually works cross platform..